mondey_backend/src/mondey_backend/routers/users.py

```diff
@@ -152,4 +156,59 @@ def update_milestone_answer(
             session.commit()
         return milestone_answer
 
+    # Endpoints for answers to user question
+    @router.get("/user-answers/", response_model=list[UserAnswerPublic])
+    def get_current_user_answers(
+        session: SessionDep, current_active_user: CurrentActiveUserDep
+    ):
+        answers = session.exec(
+            select(UserAnswer).where(col(UserAnswer.user_id) == current_active_user.id)
+        ).all()
+
+        user_answers = [
+            UserAnswerPublic(
+                answer=answer.answer,
+                non_standard=answer.non_standard,
+                question_id=answer.question_id,
+            )
+            for answer in answers
+        ]
+        return user_answers
```

Comment:
this shouldn't be required - you can just return answers and fastapi will return a list[UserAnswerPublic] as you specified in the reponse_model

Reply:
ok, changed that.